assets/chat/js/chat.js

```diff
@@ -683,6 +680,11 @@ class Chat {
     $(document.body).toggleClass(`pref-fontscale`, fontscale !== 'auto');
     $(document.body).attr('data-fontscale', fontscale);
 
+    // Add command autocomplete
+    this.commands
```

Comment:
I was able to reproduce the issue, but this feels like the wrong place. You're adding the commands to autocomplete every time a setting is changed, which doesn't make sense. Maybe in `setUser()` fits better?

Reply:
You're right, that makes a bit more sense.